src/main/java/com/google/api/codegen/transformer/SampleTransformer.java

```diff
@@ -251,6 +252,17 @@ private MethodSampleView generateSample(
                     initCodeContext.cloneWithEmptySymbolTable())); // to avoid symbol collision
     ImportSectionView sampleImportSectionView =
         sampleImportTransformer().generateImportSection(methodContext);
+    SampleFunctionDocView docView =
+        SampleFunctionDocView.newBuilder()
+            .parameters(
+                initCodeView
+                    .argDefaultParams()
+                    .stream()
+                    .filter(p -> !p.description().isEmpty())
+                    .collect(ImmutableList.toImmutableList()))
+            .description(valueSet.getDescription())
```

Comment:
The value set description was not originally intended to be the method description. It was intended as a description visible in the UI of APIx samples. However, I think it can double as a method description: the information both descriptions convey is the same, as I think about it, so this field can be the source of truth for both.

Reply:
SGTM